share/sphinx/xtp-tutorials/XTP-TUTORIAL.rst

```diff
@@ -3,7 +3,6 @@ XTP Tutorials
 
 .. toctree::
    DFTGWBSE_ENERGY
-   PYXTP
```

Comment:
@NicoRenaud how can we build the pyxtp tutorial? Did it ever work though CMake?

@baumeier is it ok to drop this for now, it never worked!

Reply:
I think the manual worked, can't remember any tutorial.

Reply:
There are some tutorials but I don't think we be ever build them in the cmake. The Python files of the tutorial are still there so we can always fall back on that for people who would want to use pyxtp

Reply:
Ok, I can help with adding to cmake, but the `xtp-tutorials/pyxtp` folder has no `ipynb`. what am I even suppose to run? 
```
$ ls xtp-tutorials/pyxtp/
files_examples  __init__.py  run_energy.py  run_gradient.py  run_mapchecker.py  run_optimization.py
```

Reply:
I can turn the .py into .ipynb if that helps ? It shouldn't take too much time

Reply:
I've made the notebooks but I think I can push directly to the repo. I can push to my own fork and make a PR from there. Or do you prefer to give me push access so that I can push my branch directly here ?

Reply:
I've pushed the notebooks to my branch https://github.com/NicoRenaud/votca/tree/xtp_tutorial

Reply:
Please make a PR!

Reply:
Oh Sorry ! just made one: #1090 Let's pick up the discussion there :)